contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,635 @@
+# Raw Block Consumption in Kubernetes
+
+Authors: erinboyd@, screeley44@, mtanino@
+
+This document presents a proposal for managing raw block storage in Kubernetes using the persistent volume source API as a consistent model of consumption.
+
+# Terminology
+* Raw Block Device - a physically attached device devoid of a filesystem
+* Raw Block Volume - a logical abstraction of the raw block device as defined by a path
+* File on Block/Filesystem on Block - a formatted (ie xfs) filesystem on top of a raw block device
+
+# Goals
+* Enable durable access to block storage
+* Support storage requirements for all workloads supported by Kubernetes
+* Provide flexibility for users/vendors to utilize various types of storage devices
+* Agree on API changes for block
+* Provide a consistent security model for block devices 
+* Provide a means for running containerized block storage offerings as non-privileged container
```

Comment:
Can you be specific as to which operations can be done with and without privileges?

Reply:
All of them. The CRI doesn't require priv to do block

Reply:
Just checked with @mtanino .  It looks like read, write, format, partition doesn't require privileges.  But mount does require privileges.  So we should call that out.

Reply:
done